scripts/build.py

```diff
@@ -1097,6 +1097,9 @@ def generate_build_directory():
 
 
 def build():
+    """Creates a third party directory where all the dependencies of JS and
+    CSS files minifies and stored.
```

Comment:
Change to -> `... Html files are stored.`. I gave this review comment in the previous PR but isn't addressed yet.

Reply:
But sir in code Css files are minified not Html . That why i write this . Thanks.

Reply:
Yup, I agree that CSS files are minified. In that case, please correct the tense of the sentence. May be- '... the dependencies of JS and CSS files are minified and stored.'

Reply:
Ok sir